convert2rhel/systeminfo.py

```diff
@@ -359,6 +360,35 @@ def corresponds_to_rhel_eus_release(self):
         """
         return self.releasever in EUS_MINOR_VERSIONS
 
+    @staticmethod
+    def _get_dbus_status():
+        """
+        Check whether dbus is running.
+
+        :returns: True if dbus is running.  Otherwise False
+        """
+        running = False
+
+        if system_info.version.major <= 6:
+            # Upstart or SysVinit system
+            output, ret_code = utils.run_subprocess(["/sbin/service", "messagebus", "status"])
+            for line in output.splitlines():
+                if line.startswith("messagebus"):
+                    if "run" in line:
+                        running = True
+                        break
+        else:
+            # Systemd
+            output, ret_code = utils.run_subprocess(["/usr/bin/systemctl", "show", "-p", "ActiveState", "dbus"])
+            for line in output.splitlines():
+                if line.startswith("ActiveState="):
+                    state = line.split("=", 1)[1]
+                    if state in ("active", "reloading", "activating"):
```

Comment:
Is there any possibility for the `reloading` or `activating` status to turn to be a failure in the end? 

I mean, I don't know if that's even possible, but, let's pretend that after reload something goes wrong, would that affect us?

Reply:
There is.  I thought about implementing retry on these but got lazy.  I've now pushed a commit which will retry three times if the state is not definitive (for systemd: active, failed, inactive, or deactivating)